app/dashboard/views.py

```diff
@@ -6493,6 +6493,33 @@ def payout_bounty_v1(request, fulfillment_id):
     return JsonResponse(response)
 
 
+@csrf_exempt
+@require_POST
+def payout_tx_forwarder_v1(request, tenant):
+    '''
+        Proxy payout transaction to external node to bypass CORS restriction on browser.
+    '''
+    import requests
+
+    if tenant.upper() == 'CASPER':
+        # casper
+        url = 'http://3.142.224.108:7777/rpc'
+    else:
+        return JsonResponse({'error': 'invalid tenant'}, status=400)
+
+    data = json.loads(request.body)
+
+    try:
+        response = requests.post(url, json=data).json()
+
+        if response.get("error", False):
+            return JsonResponse({'error': response["error"]}, status=422)
+    except Exception:
+        return JsonResponse({'error': 'something went wrong!'}, status=422)
```

Comment:
Is 422 the right error code to be used for here and above ?

Reply:
yea it is... unproccessible entity... there's a 400 above for badly formed request